src/MutationFunctionHandler.js

```diff
@@ -32,28 +32,68 @@ export default class MutationFunctionHandler {
   }
 
   async createMutationExpressions(pathData, path, args) {
-    // Check if we have a valid path
-    const conditions = await path.pathExpression;
-    if (!Array.isArray(conditions))
-      throw new Error(`${pathData} has no pathExpression property`);
-    if (conditions.length < 2)
-      throw new Error(`${pathData} should at least contain a subject and a predicate`);
-
     // The arguments are the affected objects
-    const objects = await this.extractObjects(pathData, path, args);
+    // Do this first since an object input can change the conditions
+    let objects;
+    if (args.length === 1 && this.isObject(args[0])) {
+      objects = Object.keys(args[0]).map(key => {
+        let values = args[0][key];
+        // Simulate the function being called with no arguments
+        if (values === null || values === undefined)
+          values = [];
+        // Shortcut for single arguments
+        else if (!Array.isArray(values))
+          values = [values];
+        // Currently the path value is only used by the SolidDeleteFunctionHandlers in query-ldflex
+        // and needs to be updated to be the correct path there.
```

Comment:
@joachimvh Does this mean we need to update `@solid/query-ldflex`?

Reply:
This line makes it so that `@solid/query-ldflex` does not need to be updated. If you're asking if it should be updated, the answer is it depends. :P The main "problem" is that it overwrites one of the internal functions of the MutationFunctionHandler, thereby complicating things a bit (since I had to make sure everything worked with both versions of that function). But I don't know if it can be done without doing that (I didn't actually check that).